src/components/HomeView.jsx

```diff
@@ -32,59 +39,80 @@ const styles = (theme) => ({
     paddingTop: theme.spacing(4),
   },
   title: {
-    color: theme.palette.primary.main,
-    fontSize: '54px',
-    lineHeight: '66px',
-    paddingBottom: theme.spacing(8),
+    color: theme.palette.secondary.dark,
+    fontSize: '1.5rem',
+    fontWeight: 700,
+    paddingBottom: '1rem',
   },
-  wrapper: {
-    display: 'flex',
+  tabsRoot: {
+    color: '#333F52',
     fontFamily: theme.typography.fontFamily,
-    justifyContent: 'center',
-    padding: theme.spacing(4),
-    margin: theme.spacing(4),
+    height: 18,
+    fontSize: '1rem',
+    fontWeight: 600,
+    lineHeight: 18,
+    textAlign: 'center',
+  },
+  tabSelected: {
+    fontWeight: 700,
+    color: theme.palette.secondary.dark,
+    bottomBar: '6px',
   },
-  width: {
-    ...theme.mixins.containerWidth,
+  tabsIndicator: {
+    borderBottom: `6px solid ${theme.palette.secondary.main}`,
+    transition: 'none',
   },
 });
 
 class HomeView extends React.PureComponent {
   static propTypes = {
     classes: PropTypes.object.isRequired,
-    datasets: PropTypes.array.isRequired,
-    dispatch: PropTypes.func.isRequired,
-    features: PropTypes.object,
-    snapshots: PropTypes.array.isRequired,
+    location: PropTypes.object,
   };
 
-  componentDidMount() {
-    const { dispatch } = this.props;
-    dispatch(getDatasets(5));
-    dispatch(getSnapshots(5));
-  }
+  static prefixMatcher = new RegExp('/[^/]*');
 
   render() {
-    const { classes, datasets, features, snapshots } = this.props;
+    const { classes, location } = this.props;
+    const tabValue = HomeView.prefixMatcher.exec(location.pathname)[0];
```

Comment:
Does this make sense as part of a component? I'm also surprised this isn't something that's part of the router

Reply:
I think if we want the tabs in the HomeView then we would need to handle the values here in some way, this is getting the location from the global state but I could try passing it through as a prop from the router if you think that might be a better approach?